rasa/nlu/training_data/message.py

```diff
@@ -24,16 +26,16 @@
 class Message:
     def __init__(
         self,
-        text: Text,
+        text: Optional[Text] = "",
```

Comment:
can it actually be  `None`?

Reply:
No, because it breaks reading from `json`, tries it before :)

Reply:
@evgeniiaraz I thought we agreed to remove `Message.text`

Reply:
`Message.text`  as a property is gone, so, you can't do `self.text` or anything. 
In the `__init__` I left it in because in many(many many many) places `Message` is not created through `build ` but through direct `Message(text, bla bla)`.

Reply:
@evgeniiaraz could you give examples of these places? How hard is it to update these places with passing `data{TEXT: text}`. keeping `text` in `init` seems to me as a bad solution

Reply:
Done in #6394